bgpd/bgp_evpn_mh.c

```diff
@@ -383,11 +383,12 @@ int bgp_evpn_mh_route_update(struct bgp *bgp, struct bgp_evpn_es *es,
 	 * in the VNI-rt-table.
 	 */
 	if (remote_pi) {
+		struct in_addr vtep_ip = {.s_addr = 0};
 		flog_err(
 			EC_BGP_ES_INVALID,
-			"%u ERROR: local es route for ESI: %s Vtep %pI4 also learnt from remote",
+			"%u ERROR: local es route for ESI: %s vtep %pI4 also learnt from remote",
```

Comment:
I guess we should print `(null)` (which is handled by `%pI4` instead of `vtep 0.0.0.0`, right?

Reply:
Yes.

Reply:
Thanks for help, right way. Done.